capstone-app/package.json

```diff
@@ -3,14 +3,43 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.10.0",
+    "@material-ui/icons": "^4.9.1",
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "classnames": "2.2.6",
+    "fibers": "^5.0.0",
+    "history": "4.10.1",
+    "moment": "2.26.0",
+    "node-sass": "^4.14.1",
+    "nouislider": "^14.6.0",
+    "prop-types": "15.7.2",
     "react": "^16.13.1",
+    "react-datetime": "2.16.3",
     "react-dom": "^16.13.1",
-    "react-scripts": "3.4.1",
+    "react-router-dom": "5.2.0",
+    "react-scripts": "^3.4.1",
+    "react-slick": "0.26.1",
+    "react-swipeable-views": "0.13.9",
+    "sass": "^1.26.10",
     "serve": "^11.3.2"
   },
+  "devDependencies": {
+    "@babel/cli": "7.10.1",
+    "@babel/plugin-proposal-class-properties": "7.10.1",
+    "@babel/preset-env": "7.10.1",
+    "@babel/preset-react": "7.10.1",
+    "eslint-config-prettier": "6.11.0",
+    "eslint-plugin-prettier": "3.1.3",
+    "eslint-plugin-react": "7.20.0",
+    "gulp": "4.0.2",
+    "gulp-append-prepend": "1.0.8",
+    "prettier": "2.0.5"
+  },
+  "optionalDependencies": {
+    "typescript": "3.9.3"
```

Comment:
nit - is there typescript in the project?

Reply:
It's an optional dependency for one of the other dependencies